samples/grpc/shopping-cart-service-scala/k8s/deployment.yaml

```diff
@@ -54,7 +54,7 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: JAVA_TOOL_OPTIONS
-              value: "-XX:InitialRAMPercentage=75 -XX:MaxRAMPercentage=75 -Dshopping-cart-service.grpc.port=8080 -Dakka.persistence.r2dbc.connection-factory.host=10.8.205.4 -Dakka.persistence.r2dbc.connection-factory.ssl.enabled=on -Dakka.persistence.r2dbc.connection-factory.ssl.mode=require"
+              value: "-XX:InitialRAMPercentage=75 -XX:MaxRAMPercentage=75 -Dshopping-cart-service.grpc.port=8080 -Dakka.persistence.r2dbc.connection-factory.host=10.8.205.4 -Dakka.persistence.r2dbc.connection-factory.ssl.enabled=on -Dakka.persistence.r2dbc.connection-factory.ssl.mode=require -Dshopping-cart-service.simulator-count=10 -Dshopping-cart-service.simulator-delay=200ms"
```

Comment:
Modify the throughput load here when deploying. Be aware of that it starts one simulator per pod, so 4x if you run 4 pods.
```
-Dshopping-cart-service.simulator-count=10 -Dshopping-cart-service.simulator-delay=200ms
```